nRF9160Master/src/main.c

```diff
@@ -1300,6 +1302,7 @@ static void SystemTask()
 #endif
 	while (1)
 	{   
+		calculate_Temperature(); //called to get the temperature 
```

Comment:
Lets use the temperature in the payload for Wifi, BLE and LTE. @DevenduRajan

Reply:
@DevenduRajan Please commit the mentioned changes ASAP tomorrow. Divin had made some changes for SSID and password related stuff. He can only commit  his change after you commit your changes.  @divin-raj-kp After this changes are committed, just stash the changes you already done.  Pull the changes from repo. and apply stash to update your changes. and then push it. If you have any doubt please ping me before you do it.

Reply:
changes are done as per the mentions

Reply:
@divin-raj-kp I'm merging this branch. After that you can pull the changes from this one and commit your changes